src/logic_analyzer.cpp

```diff
@@ -710,8 +710,11 @@ QString LogicAnalyzer::saveToFile()
 	    &selectedFilter, (m_useNativeDialogs ? QFileDialog::Options() : QFileDialog::DontUseNativeDialog));
 
 
-	if(fileName.isEmpty())
+	if(fileName.isEmpty()) {
```

Comment:
curios: is there a preferred style here with respect to `if()` vs `if ()`
the file does not seem consistent, so i don't know which should be used;